x-pack/plugins/lens/public/xy_visualization/expression.tsx

```diff
@@ -380,30 +381,14 @@ export function XYChart({
     return style;
   };
 
-  return (
-    <Chart>
-      <Settings
-        showLegend={
-          legend.isVisible && !legend.showSingleSeries
-            ? chartHasMoreThanOneSeries
-            : legend.isVisible
-        }
-        legendPosition={legend.position}
-        showLegendExtra={false}
-        theme={chartTheme}
-        baseTheme={chartBaseTheme}
-        tooltip={{
-          headerFormatter: (d) => safeXAccessorLabelRenderer(d.value),
-        }}
-        rotation={shouldRotate ? 90 : 0}
-        xDomain={xDomain}
-        onBrushEnd={({ x }) => {
+  const extraSettings: Partial<SettingsSpec> = isTimeViz
+    ? {
+        onBrushEnd: ({ x }) => {
```

Comment:
This function is entirely copy+pasted and moved. The diff shows it being a larger change than it was.